nextcord/types/voice.py

```diff
@@ -7,7 +7,12 @@
 from .member import MemberWithUser
 from .snowflake import Snowflake
 
-SupportedModes = Literal["xsalsa20_poly1305_lite", "xsalsa20_poly1305_suffix", "xsalsa20_poly1305"]
+SupportedModes = Literal[
+    "aead_xchacha20_poly1305_rtpsize",
+    "xsalsa20_poly1305_lite",
```

Comment:
We might as well get rid of these as the documentation claims you can no longer use these to connect.